src/gui/mod.rs

```diff
@@ -1043,26 +1036,38 @@ impl Application for Ajour {
                             None
                         }
                     }
+                    Some(State::Error(error)) => {
+                        let error_title = format!("{}", error);
+                        let mut error_description = String::new();
+                        let cause = error.chain().last();
+
+                        if let Some(cause) = cause {
+                            error_description.push_str(&format!("Caused by {}", cause)[..]);
+                        }
+
+                        Some(element::status::data_container(
+                            color_palette,
+                            &error_title,
+                            &error_description,
+                            None,
+                        ))
+                    }
+                    None => Some(element::status::data_container(
+                        color_palette,
+                        &localized_string("setup-ajour-title")[..],
+                        &localized_string("setup-ajour-description")[..],
+                        Some(&mut self.onboarding_directory_btn_state),
+                    )),
```

Comment:
Before we used `State::Start` to know if it was "first time" but I figured it was better just to check if we had anything in our state HashMap and utilize the fact that if there's nothing in it, it must mean we are in an "Start" state.